compute_endpoint/globus_compute_endpoint/cli.py

```diff
@@ -312,6 +313,25 @@ def start_endpoint(*, ep_dir: pathlib.Path, **_kwargs):
     )
 
 
+@app.command(name="login", help="Manually log in with Globus")
+@click.option(
+    "--force",
+    is_flag=True,
+    help="Log in even if already logged in",
+)
+def login(force: bool):
+    if is_client_login():
+        log.info("Don't need to log in when client credentials are present")
+        sys.exit(0)
+
```

Comment:
Nice ... do we want to attempt a trivial communication with those credentials to verify they work?

Reply:
I'm gonna say "maybe later," because we don't do any actual validation for non-client credentials either, and validating these credentials is a much bigger ask than just checking if they're there